### PR TITLE
Add "stable" version alias to server bootstrap endpoint

### PR DESCRIPTION
**Pull request by iammattcoleman**

### CODE REVIEW

File: src/main/java/net/fabricmc/meta/web/ServerBootstrap.java [hunk 1]
```diff
@@ -75,6 +76,10 @@ private static Handler boostrapHandler() {
         String version = ctx.pathParam(name);
 
         for (V v : versions) {
+            if (version.equals("stable") && v.isStable()) {
```
Comment (modmuss50) [inline src/main/java/net/fabricmc/meta/web/ServerBootstrap.java:79]: I think we said this shouldnt work for the minecraft version.

**Review by modmuss50 — CHANGES_REQUESTED** [review]

File: src/main/java/net/fabricmc/meta/web/ServerBootstrap.java (hunk 1 above)
Comment (iammattcoleman) [reply src/main/java/net/fabricmc/meta/web/ServerBootstrap.java:79]: I brought this up on Discord but didn't tag you, sorry.

That's easily doable here by adding ` && !name.equals("game_version")` to the conditional, but why not leave the code simpler and let the admin dig their own grave if they want to auto-update the Minecraft version?

Comment (modmuss50) [reply src/main/java/net/fabricmc/meta/web/ServerBootstrap.java:79]: Fair enough. 👍

File: src/main/java/net/fabricmc/meta/web/ServerBootstrap.java [hunk 2]
```diff
@@ -177,4 +182,11 @@ private static void writePropertiesToJar(Path inputJar, Path outputJar, String l
         Files.copy(workingFile, outputJar);
         Files.delete(workingFile);
     }
+
+    private static int getCacheDuration(Context ctx) {
+        if (ctx.pathParamMap().containsValue("stable")) {
+            return 900;
```
Comment (sfPlayer1) [inline src/main/java/net/fabricmc/meta/web/ServerBootstrap.java:188]: I think 900 is still too high, people may expect less after a MC release/snapshot? Maybe 120?

Comment (iammattcoleman) [reply src/main/java/net/fabricmc/meta/web/ServerBootstrap.java:188]: The two cache duration values that were suggested to me on Discord were 15 minutes and 1 hour. I opted for the shorter one. I agree that "the shorter the better" is the way to go, but don't want to do anything that would hammer the server too much.

**Review by modmuss50 — APPROVED** [review]
LGTM. I want to test this before releaseing.

Thanks 👍

**Review by sfPlayer1 — APPROVED** [review]

**Comment by iammattcoleman** [comment]
@modmuss50 Here's a script you can use to test this change in a local instance:
https://gist.github.com/iammattcoleman/864377c5e92db5abf0a07e68618675f0

**Pull request merged**